lib/models/patient/resources.js

```diff
@@ -107,6 +110,10 @@ module.exports = function (PatientSchema) {
         }
     }, {});
 
+    PatientSchema.methods.queryEvents = query("events", Event,{
+        // what goes in this function
```

Comment:
That object contains parameters for authorizing access to a specific record. Because we're not doing any specific authorization here (other than requiring access to the patient in question), we can just leave it empty. Can you update by either adding a comment clarifying, or remove the "what goes in this function" comment (for code cleanliness)? Thanks!